Docs/Configuration.tex

```diff
@@ -485,54 +496,53 @@ \subsection{Directory Structure}\label{directory-structure}
 Figure 1. Directory Structure
 \end{center}
 
-When directory boot is used the directory structure used should follow
-the description on \hyperref[fig:DS]{Directory Structure} figure. Available
-entries include:
+When directory boot is used, the directory structure used should follow
+the descriptions in the \hyperref[fig:DS]{Directory Structure} figure.
+Available entries include:
 
 \begin{itemize}
 \tightlist
 \item
   \texttt{BOOTx64.efi} or \texttt{BOOTIa32.efi} \\
   Initial bootstrap loaders, which load \texttt{OpenCore.efi}. \texttt{BOOTx64.efi}
-  is loaded by the firmware by default according to UEFI specification, yet it can
-  also be renamed and put to a custom location to let OpenCore coexist with operating
-  systems using \texttt{BOOTx64.efi} as their own loaders (e.g. Windows), see
-  \texttt{LauncherOption} for more details.
+  is loaded by the firmware by default according to UEFI specifications. However, it can
```

Comment:
It is just one spec. maybe `to the UEFI specification`.

Reply:
Should always be "specifications" when talking about the entire set.
Each individual item they have specified is a specification.

Reply:
Would agree with @dakanji about that usage. Could change to "**the** UEFI specifications" in the text? Also correct, but more clear that there's only one set.

Reply:
Incorrect. It is UEFI specifications when talking about them all (UEFI, UEFI PI, ACPI, etc), but UEFI specification when referring to the particular document. See https://uefi.org/specifications

Reply:
> It is `UEFI specifications` when talking about them all, but `UEFI specification` when referring to the particular document.

Well, they are abusing the language but yes, they have indeed done that to get the result you outlined.

Each one should not have been singular and the singular one should have been called `Something Else Specifications` so that the whole could remain as `UEFI specifications` but I suppose they couldn't find a suitable name and decided on that approach.

It is what it is and I understand the aim here to stay aligned. Will change accordingly.